plugins/generator-1.16.5/forge-1.16.5/utils/boundingboxes.java.ftl

```diff
@@ -1,30 +1,47 @@
-<#macro makeBoundingBox positiveBoxes negativeBoxes noOffset facing>
+<#macro makeBoundingBox positiveBoxes negativeBoxes noOffset facing pitchType="wall">

```

Comment:
The default value of pitchType here should be `floor`, but I forgot to change it when editing bounding boxes last time and it's now `wall`, so normal blocks are gonna be broken. How could you not notice this?

Reply:
Same way you didn't notice the non-opaque checkbox. My bad. You can open a PR to fix the issue and I'll merge it

Reply:
Unlike me, you have write permissions.

Reply:
I don't have write permissions, I can't edit the main branch, only merge approved requests

Reply:
@Defeatomizer, it needs to be done through PR as SomeoneElse said

Reply:
> Same way you didn't notice the non-opaque checkbox. 

Was this part also resolved?

Reply:
>@Defeatomizer, it needs to be done through PR as SomeoneElse said

The PR was merged 23 hours before you said this.

Reply:
I was talking for future reference